rules.ini

```
@@ -133,6 +133,7 @@ PlayFirstPlayground = \.pfp$
 Prism3D[] = (?:^|/)base\.scs$
 Prism3D[] = (?:^|/)p3shared\.dll$
 PyGame = (?:^|/)pygame
+raylib = (?:^|/)(?:lib)?raylib\.(?:dll|so)$
```

```suggestion
raylib = (?:^|/)(?:lib)?raylib\.
```
This should detect 1 more app(https://steamdb.info/app/1661930/depots/) 
although it's still not a lot

---

Oh! That's nice! Thanks for the improvement!